docs/reference/commandline/service_create.md

```diff
@@ -320,9 +320,20 @@ volumes in a service:
       </ul></p>
     </td>
   </tr>
+  <tr>
+    <td><b>consistency</b></td>
+    <td></td>
+    <td>
+      <p>The consistency requirements for the mount:
+         <tt>default</tt>,
```

Comment:
Thanks for the review, @mstanleyjones.  I've added a `<ul>` explaining the modes, based on @andrerom's suggestion.